service/lexruntimeservice/deserializers.go

```diff
@@ -579,6 +715,31 @@ func awsRestjson1_deserializeOpDocumentGetSessionOutput(v *GetSessionOutput, dec
 			}
 			v.SessionId = st
 
+		case "slotToElicit":
+			val, err := decoder.Token()
+			if err != nil {
+				return false, err
+			}
+			var st *string
+			if val != nil {
+				jtv, ok := val.(string)
+				if !ok {
+					return false, fmt.Errorf("expected SlotToElicit to be of type string, got %T instead", val)
+				}
+				st = &jtv
+			}
+			v.SlotToElicit = st
+
+		case "slots":
+			col := map[string]*string{}
+			if present, err := awsRestjson1_deserializeDocumentStringMap(&col, decoder); err != nil {
+				return false, err
+			} else if !present {
+				col = nil
```

Comment:
will want to rebase with latest version of sdk. i think these generation was updated to do nil checks here before entering the deserializaiton function.